index.bs

```diff
@@ -874,6 +874,10 @@ An attribution report is a [=struct=] with the following items:
 
 </dl>
 
+Issue: Consider removing the [=attribution report/delivered=] field, and instead
```

Comment:
This is probably needed for the retry logic.

Reply:
I'm not sure if it's necessary for that, but even if it is, the value of that field should probably be asserted to be false in various other algorithms.